isanteplusreportsddlscript.sql

```diff
@@ -944,4 +944,55 @@ CREATE TABLE IF NOT EXISTS patient_ob_gyn (
 );
 /* <end ob/gyn>*/
 
-
+	/*Create table for art reort*/
+	DROP TABLE IF EXISTS patient_on_art;
+	CREATE TABLE IF NOT EXISTS patient_on_art(
+	patient_id int(11),
+	encounter_id int(11),
+	visit_date date,
+	visit_id int(11),
+	location_id int(11),
+	encounter_type int(11) DEFAULT NULL,
+	enrolled_on_art int(11) DEFAULT NULL,
+	gender varchar(10) DEFAULT NULL,
+	key_population VARCHAR(255),
+	breast_feeding int(11) DEFAULT NULL,
+	tested_hiv_postive int(11) DEFAULT NULL,
+	reason_non_enrollemnt VARCHAR(255),
+	enrolled_tb_treatment int(11) DEFAULT NULL,
+	transferred int(11) DEFAULT NULL,
+	tb_screen int(11) DEFAULT NULL,
+   tb_status varchar(10) DEFAULT NULL,
+   tb_RIF_test int(11) DEFAULT NULL,
+   tb_other_test int(11) DEFAULT NULL,
+   started_anti_tb_treatment int(11) DEFAULT NULL,
+   date_started_anti_tb_treatment DATETIME,
+   tb_bacteriological_test_status varchar(10) DEFAULT NULL,
+   lost int(11) DEFAULT NULL,
+   date_inactive  DATETIME ,
+   inactive_reason VARCHAR(20) DEFAULT NULL,
+   inactive int(11) DEFAULT NULL,
+   deceased int(11) DEFAULT NULL,
+   recieve_arv int(11) DEFAULT NULL,
+   date_started_arv DATETIME,
+   date_started_recieving_arv DATETIME,
+   recieve_clinical_followup int(11) DEFAULT NULL,
+   treatment_regime_lines varchar(10) DEFAULT NULL,
+   lost_reason varchar(10) DEFAULT NULL,
+   date_lost DATETIME,
+   period_lost varchar(10) DEFAULT NULL,
+   cause_of_death_for_lost varchar(10) DEFAULT NULL,
+   viral_load_result varchar(10) DEFAULT NULL,
+   viral_load_targeted varchar(10) DEFAULT NULL,
+   resumed_arv_after_lost int(11) DEFAULT NULL,
+   cervical_cancer_screenee int(11) DEFAULT NULL,
+   cervical_cancer_status varchar(10) DEFAULT NULL,
+   cervical_cancer_screenee_after_first_time int(11) DEFAULT NULL,
+   cervical_cancer_screenee_after_first_treat int(11) DEFAULT NULL,
```

Comment:
Change the text `screnee` to  `screening`

Reply:
corrected